packages/apps/nats/values.yaml

```diff
@@ -4,7 +4,11 @@
 ## @param external Enable external access from outside the cluster
 ## @param replicas Persistent Volume size for NATS
 ## @param storageClass StorageClass used to store the data
+## @param nats username, if empty no auth
+## @param nats password, if empty no auth
 ##
 external: false
 replicas: 2
 storageClass: ""
+username:
+password:
```

Comment:
Let's keep the format we're using in other apps like postgres, mysql, rabbitmq, vpn and so on:

https://github.com/aenix-io/cozystack/blob/47bd46c17123cbef515480a4e1d9c1224cc5f715/packages/apps/postgres/values.yaml#L28-L36